app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -195,7 +195,7 @@
         <div class="ui action input fluid">
           <input type="text" name="private_link" value="{{concat this.data.event.url '/cfs/' this.data.speakersCall.hash}}" placeholder="{{concat this.data.event '/cfs/' this.data.speakersCall.hash}}" readonly>
           <UiPopup @on="click" @content={{t "Link copied to clipboard"}} @position="bottom right">
-            <CopyButton @clipboardText={{concat this.data.event.url "/cfs/" this.data.speakersCall.hash}} @class="ui black right labeled icon copy button">
+            <CopyButton @clipboardText={{concat this.data.event.url "/cfs/" this.data.speakersCall.hash}} @class="ui black right labeled icon copy button" style="height: 100%;">
```

Comment:
I see more than 1 of your PRs is using style for `height: 100%`.
Please create a class instead and use it